plugin.py

```diff
@@ -17,6 +19,8 @@
 TAG = "0.7.3"
 GOPLS_BASE_URL = 'golang.org/x/tools/gopls@v{tag}'
 
+RE_VER = re.compile(r"go(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)")
```

Comment:
What's the logic behind this strange regex?

why not just:
```suggestion
RE_VER = re.compile(r"go(\d+)\.(\d+)\.(\d+)")
```
?